{
  "uid" : "db1dd3fe7d1b722b",
  "name" : "Verify presence, visibility of the title, tiles on the page",
  "fullName" : "tests.groups_page_test.TestGroupsPageStructure#test_gp_01_03_verify_page_structural_elements",
  "historyId" : "d85cff542f7f0e433fd329408086cbff",
  "time" : {
    "start" : 1728781238508,
    "stop" : 1728781238508,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55de3c2a002a <unknown>\n#1 0x55de3bf865e0 <unknown>\n#2 0x55de3bf7e7eb <unknown>\n#3 0x55de3bf6ea09 <unknown>\n#4 0x55de3bf70744 <unknown>\n#5 0x55de3bf6ecbd <unknown>\n#6 0x55de3bf6e52f <unknown>\n#7 0x55de3bf6e422 <unknown>\n#8 0x55de3bf6c493 <unknown>\n#9 0x55de3bf6cada <unknown>\n#10 0x55de3bf88f41 <unknown>\n#11 0x55de3c01a8c5 <unknown>\n#12 0x55de3bffab22 <unknown>\n#13 0x55de3c019d7d <unknown>\n#14 0x55de3bffa8c3 <unknown>\n#15 0x55de3bfc86b3 <unknown>\n#16 0x55de3bfc968e <unknown>\n#17 0x55de3c26aa2b <unknown>\n#18 0x55de3c26e9b1 <unknown>\n#19 0x55de3c257225 <unknown>\n#20 0x55de3c26f532 <unknown>\n#21 0x55de3c23c38f <unknown>\n#22 0x55de3c28ef28 <unknown>\n#23 0x55de3c28f0f3 <unknown>\n#24 0x55de3c29ee7c <unknown>\n#25 0x7f999ac94ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"da533169a7c7d5dcea995c475bb7b541\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fe0c1168590>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x55de3c28f0f3 \\\\u003Cunknown>\\\\n#24 0x55de3c29ee7c \\\\u003Cunknown>\\\\n#25 0x7f999ac94ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x55de3c2a002a <unknown>\nE       #1 0x55de3bf865e0 <unknown>\nE       #2 0x55de3bf7e7eb <unknown>\nE       #3 0x55de3bf6ea09 <unknown>\nE       #4 0x55de3bf70744 <unknown>\nE       #5 0x55de3bf6ecbd <unknown>\nE       #6 0x55de3bf6e52f <unknown>\nE       #7 0x55de3bf6e422 <unknown>\nE       #8 0x55de3bf6c493 <unknown>\nE       #9 0x55de3bf6cada <unknown>\nE       #10 0x55de3bf88f41 <unknown>\nE       #11 0x55de3c01a8c5 <unknown>\nE       #12 0x55de3bffab22 <unknown>\nE       #13 0x55de3c019d7d <unknown>\nE       #14 0x55de3bffa8c3 <unknown>\nE       #15 0x55de3bfc86b3 <unknown>\nE       #16 0x55de3bfc968e <unknown>\nE       #17 0x55de3c26aa2b <unknown>\nE       #18 0x55de3c26e9b1 <unknown>\nE       #19 0x55de3c257225 <unknown>\nE       #20 0x55de3c26f532 <unknown>\nE       #21 0x55de3c23c38f <unknown>\nE       #22 0x55de3c28ef28 <unknown>\nE       #23 0x55de3c28f0f3 <unknown>\nE       #24 0x55de3c29ee7c <unknown>\nE       #25 0x7f999ac94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728781182641,
      "stop" : 1728781182641,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728781239284,
      "stop" : 1728781239875,
      "duration" : 591
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55de3c2a002a <unknown>\n#1 0x55de3bf865e0 <unknown>\n#2 0x55de3bf7e7eb <unknown>\n#3 0x55de3bf6ea09 <unknown>\n#4 0x55de3bf70744 <unknown>\n#5 0x55de3bf6ecbd <unknown>\n#6 0x55de3bf6e52f <unknown>\n#7 0x55de3bf6e422 <unknown>\n#8 0x55de3bf6c493 <unknown>\n#9 0x55de3bf6cada <unknown>\n#10 0x55de3bf88f41 <unknown>\n#11 0x55de3c01a8c5 <unknown>\n#12 0x55de3bffab22 <unknown>\n#13 0x55de3c019d7d <unknown>\n#14 0x55de3bffa8c3 <unknown>\n#15 0x55de3bfc86b3 <unknown>\n#16 0x55de3bfc968e <unknown>\n#17 0x55de3c26aa2b <unknown>\n#18 0x55de3c26e9b1 <unknown>\n#19 0x55de3c257225 <unknown>\n#20 0x55de3c26f532 <unknown>\n#21 0x55de3c23c38f <unknown>\n#22 0x55de3c28ef28 <unknown>\n#23 0x55de3c28f0f3 <unknown>\n#24 0x55de3c29ee7c <unknown>\n#25 0x7f999ac94ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728781239284,
        "stop" : 1728781239874,
        "duration" : 590
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55de3c2a002a <unknown>\n#1 0x55de3bf865e0 <unknown>\n#2 0x55de3bf7e7eb <unknown>\n#3 0x55de3bf6ea09 <unknown>\n#4 0x55de3bf70744 <unknown>\n#5 0x55de3bf6ecbd <unknown>\n#6 0x55de3bf6e52f <unknown>\n#7 0x55de3bf6e422 <unknown>\n#8 0x55de3bf6c493 <unknown>\n#9 0x55de3bf6cada <unknown>\n#10 0x55de3bf88f41 <unknown>\n#11 0x55de3c01a8c5 <unknown>\n#12 0x55de3bffab22 <unknown>\n#13 0x55de3c019d7d <unknown>\n#14 0x55de3bffa8c3 <unknown>\n#15 0x55de3bfc86b3 <unknown>\n#16 0x55de3bfc968e <unknown>\n#17 0x55de3c26aa2b <unknown>\n#18 0x55de3c26e9b1 <unknown>\n#19 0x55de3c257225 <unknown>\n#20 0x55de3c26f532 <unknown>\n#21 0x55de3c23c38f <unknown>\n#22 0x55de3c28ef28 <unknown>\n#23 0x55de3c28f0f3 <unknown>\n#24 0x55de3c29ee7c <unknown>\n#25 0x7f999ac94ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"da533169a7c7d5dcea995c475bb7b541\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728781238508,
      "stop" : 1728781239284,
      "duration" : 776
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728781239907,
      "stop" : 1728781239969,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "The Groups Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "groups_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestGroupsPage"
  }, {
    "name" : "host",
    "value" : "fv-az585-604"
  }, {
    "name" : "thread",
    "value" : "2289-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.groups_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 2,
        "unknown" : 0,
        "total" : 5
      },
      "items" : [ {
        "uid" : "9ab4db306d3c6170",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/150//#testresult/9ab4db306d3c6170",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55cd4ef6c02a <unknown>\n#1 0x55cd4ec525e0 <unknown>\n#2 0x55cd4ec4a7eb <unknown>\n#3 0x55cd4ec3aa09 <unknown>\n#4 0x55cd4ec3c744 <unknown>\n#5 0x55cd4ec3acbd <unknown>\n#6 0x55cd4ec3a52f <unknown>\n#7 0x55cd4ec3a422 <unknown>\n#8 0x55cd4ec38493 <unknown>\n#9 0x55cd4ec38ada <unknown>\n#10 0x55cd4ec54f41 <unknown>\n#11 0x55cd4ece68c5 <unknown>\n#12 0x55cd4ecc6b22 <unknown>\n#13 0x55cd4ece5d7d <unknown>\n#14 0x55cd4ecc68c3 <unknown>\n#15 0x55cd4ec946b3 <unknown>\n#16 0x55cd4ec9568e <unknown>\n#17 0x55cd4ef36a2b <unknown>\n#18 0x55cd4ef3a9b1 <unknown>\n#19 0x55cd4ef23225 <unknown>\n#20 0x55cd4ef3b532 <unknown>\n#21 0x55cd4ef0838f <unknown>\n#22 0x55cd4ef5af28 <unknown>\n#23 0x55cd4ef5b0f3 <unknown>\n#24 0x55cd4ef6ae7c <unknown>\n#25 0x7f09e0e94ac3 <unknown>",
        "time" : {
          "start" : 1728694561730,
          "stop" : 1728694561730,
          "duration" : 0
        }
      }, {
        "uid" : "a52a6cda87bbd09c",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/149//#testresult/a52a6cda87bbd09c",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56065f49f02a <unknown>\n#1 0x56065f1855e0 <unknown>\n#2 0x56065f17d7eb <unknown>\n#3 0x56065f16da09 <unknown>\n#4 0x56065f16f744 <unknown>\n#5 0x56065f16dcbd <unknown>\n#6 0x56065f16d52f <unknown>\n#7 0x56065f16d422 <unknown>\n#8 0x56065f16b493 <unknown>\n#9 0x56065f16bada <unknown>\n#10 0x56065f187f41 <unknown>\n#11 0x56065f2198c5 <unknown>\n#12 0x56065f1f9b22 <unknown>\n#13 0x56065f218d7d <unknown>\n#14 0x56065f1f98c3 <unknown>\n#15 0x56065f1c76b3 <unknown>\n#16 0x56065f1c868e <unknown>\n#17 0x56065f469a2b <unknown>\n#18 0x56065f46d9b1 <unknown>\n#19 0x56065f456225 <unknown>\n#20 0x56065f46e532 <unknown>\n#21 0x56065f43b38f <unknown>\n#22 0x56065f48df28 <unknown>\n#23 0x56065f48e0f3 <unknown>\n#24 0x56065f49de7c <unknown>\n#25 0x7fc3f7a94ac3 <unknown>",
        "time" : {
          "start" : 1728608175847,
          "stop" : 1728608175847,
          "duration" : 0
        }
      }, {
        "uid" : "818cd751ec5e6347",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/322//#testresult/818cd751ec5e6347",
        "status" : "passed",
        "time" : {
          "start" : 1728556917782,
          "stop" : 1728556917833,
          "duration" : 51
        }
      }, {
        "uid" : "b8b068a33ef2a68f",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/321//#testresult/b8b068a33ef2a68f",
        "status" : "passed",
        "time" : {
          "start" : 1728553840191,
          "stop" : 1728553840257,
          "duration" : 66
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "db1dd3fe7d1b722b.json",
  "parameterValues" : [ ]
}